docs/reference/commandline/cli.md

```diff
@@ -5,13 +5,13 @@ keywords: "Docker, Docker documentation, CLI, command line, config.json, CLI con
 aliases:
   - /reference/commandline/cli/
   - /engine/reference/commandline/engine/
-  - /engine/reference/commandline/engine_activate/
-  - /engine/reference/commandline/engine_check/
-  - /engine/reference/commandline/engine_update/
+  - /engine/reference/commandline/engine/activate/
+  - /engine/reference/commandline/engine/check/
+  - /engine/reference/commandline/engine/update/
```

Comment:
These should not be updated, as these were historic URLs

Reply:
not sure how it caught this. My python isn't what it used to be